skeletons/constr_SET_OF.c

```diff
@@ -1097,6 +1111,86 @@ SET_OF_encode_uper(const asn_TYPE_descriptor_t *td,
     }
 }
 
+asn_enc_rval_t
+SET_OF_encode_aper(const asn_TYPE_descriptor_t *td,
+                   const asn_per_constraints_t *constraints, const void *sptr,
+                   asn_per_outp_t *po) {
+    const asn_anonymous_set_ *list;
+    const asn_per_constraint_t *ct;
+    const asn_TYPE_member_t *elm = td->elements;
+    struct _el_buffer *encoded_els;
+    asn_enc_rval_t er = {0,0,0};
+
+    if(!sptr) ASN__ENCODE_FAILED;
+
+    list = _A_CSET_FROM_VOID(sptr);
+
+    er.encoded = 0;
+
+    ASN_DEBUG("Encoding %s as SET OF (%d)", td->name, list->count);
+
+    if(constraints) ct = &constraints->size;
+    else if(td->encoding_constraints.per_constraints)
+        ct = &td->encoding_constraints.per_constraints->size;
+    else ct = 0;
+
+    /* If extensible constraint, check if size is in root */
+    if(ct) {
+        int not_in_root =
+            (list->count < ct->lower_bound || list->count > ct->upper_bound);
+        ASN_DEBUG("lb %ld ub %ld %s", ct->lower_bound, ct->upper_bound,
+                  ct->flags & APC_EXTENSIBLE ? "ext" : "fix");
+        if(ct->flags & APC_EXTENSIBLE) {
+            /* Declare whether size is in extension root */
+            if(per_put_few_bits(po, not_in_root, 1)) ASN__ENCODE_FAILED;
+            if(not_in_root) ct = 0;
+        } else if(not_in_root && ct->effective_bits >= 0) {
+            ASN__ENCODE_FAILED;
+        }
+
+    }
+
+    if(ct && ct->effective_bits >= 0) {
+        /* X.691, #19.5: No length determinant */
+        /*if(per_put_few_bits(po, list->count - ct->lower_bound,
+                            ct->effective_bits))
+            ASN__ENCODE_FAILED;*/
+
+        if (aper_put_length(po, ct->upper_bound - ct->lower_bound + 1, list->count - ct->lower_bound) < 0) {
+            ASN__ENCODE_FAILED;
+	}
+    }
+
+    /*
+     * Canonical PER #22.1 mandates dynamic sorting of the SET OF elements
+     * according to their encodings. Build an array of the encoded elements.
+     */
+    encoded_els = SET_OF__encode_sorted(elm, list, SOES_CAPER);
+
+    for(int seq = 0; seq < list->count;) {
```

Comment:
Travis tests fails because of
``` 
constr_SET_OF.c: In function â€˜SET_OF_encode_aperâ€™:
constr_SET_OF.c:1170:5: error: â€˜forâ€™ loop initial declarations are only allowed in C99 mode
     for(int seq = 0; seq < list->count;) {
     ^
constr_SET_OF.c:1170:5: note: use option -std=c99 or -std=gnu99 to compile your code
```
Full log https://api.travis-ci.org/v3/job/450959466/log.txt
```suggestion
    int seq;
    for(seq = 0; seq < list->count;) {
```
As our goal is skeletons' code to be portable.